conbench/api/compare.py

```diff
@@ -169,6 +169,10 @@ def get(self, compare_ids) -> f.Response:
         baseline_benchmark_result = self._get_results(baseline_id)[0]
         contender_benchmark_result = self._get_results(contender_id)[0]
 
+        if baseline_benchmark_result.run.commit is None:
+            # TODO: emit a useful HTTP response
+            raise Exception("baseline commit unknown")
```

Comment:
Interesting outfall of type checking, mypy telling us that this condition wants to be handled.